Makefile

```diff
@@ -0,0 +1,73 @@
+SHELL ?= /bin/bash
+
+.DEFAULT_GOAL := build
+
+################################################################################
+# Version details                                                              #
+################################################################################
+
+# This will reliably return the short SHA1 of HEAD or, if the working directory
+# is dirty, will return that + "-dirty"
+GIT_VERSION = $(shell git describe --always --abbrev=7 --dirty --match=NeVeRmAtCh)
+
+################################################################################
+# Containerized development environment-- or lack thereof                      #
+################################################################################
+
+ifneq ($(SKIP_DOCKER),true)
+	PROJECT_ROOT := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+
+	KANIKO_IMAGE := brigadecore/kaniko:v0.2.0
+
+	KANIKO_DOCKER_CMD := docker run \
+		-it \
+		--rm \
+		-e SKIP_DOCKER=true \
+		-e DOCKER_PASSWORD=$${DOCKER_PASSWORD} \
+		-v $(PROJECT_ROOT):/workspaces/go-tools \
+		-w /workspaces/go-tools \
+		$(KANIKO_IMAGE)
+endif
+
+################################################################################
+# Docker images we build and publish                              #
+################################################################################
+
+ifdef DOCKER_REGISTRY
+	DOCKER_REGISTRY := $(DOCKER_REGISTRY)/
+endif
+
+ifdef DOCKER_ORG
+	DOCKER_ORG := $(DOCKER_ORG)/
+endif
+
+ifdef VERSION
+	MUTABLE_DOCKER_TAG := latest
+else
+	VERSION            := $(GIT_VERSION)
+	MUTABLE_DOCKER_TAG := edge
+endif
+
+IMMUTABLE_DOCKER_TAG := $(VERSION)
+
+.PHONY: build
+build:
+	$(KANIKO_DOCKER_CMD) kaniko \
+		--dockerfile /workspaces/go-tools/Dockerfile \
+		--context dir:///workspaces/go-tools/ \
+		--no-push
+
+################################################################################
+# Publish                                                                      #
+################################################################################
+
+.PHONY: push
+push:
+	$(KANIKO_DOCKER_CMD) sh -c ' \
+		docker login $(DOCKER_REGISTRY) -u $(DOCKER_USERNAME) -p $${DOCKER_PASSWORD} && \
+		kaniko \
+			--dockerfile /workspaces/go-tools/Dockerfile \
+			--context dir:///workspaces/go-tools/ \
+			--destination $(DOCKER_IMAGE_PREFIX)go-tools:$(IMMUTABLE_DOCKER_TAG) \
```

Comment:
It looks like the declaration of the value for DOCKER_IMAGE_PREFIX might be missing

Reply:
Thanks! Fixed!